sections/06_discussion.md

```diff
@@ -631,7 +630,7 @@ These individual representations are further concatenated before or within
 fully-connected layers. FIDDLE [@tag:Eser2016_fiddle] is an example of a
 multimodal CNN that represents an ensemble of individual networks that take
 NET-seq, MNase-seq, ChIP-seq, RNA-seq, and raw DNA sequence as input to predict
-Transcription Start Site-seq. The combined model radically improves performance
+transcription start sites. The combined model radically improves performance
```

Comment:
Works for me - previous wording was more specific but that specificity is not likely to be needed in this context.